app/views/idv/in_person/state_id.html.erb

```diff
@@ -111,10 +111,37 @@
         ) %>
   </div>
   <div class="margin-bottom-5">
+    <% state_id_number_hint_default = capture do %>
+      <%= t('in_person_proofing.form.state_id.state_id_number_hint') %>
+      <% [
+           [t('in_person_proofing.form.state_id.state_id_number_hint_spaces'), ' '],
+           [t('in_person_proofing.form.state_id.state_id_number_hint_forward_slashes'), '/'],
+           [t('in_person_proofing.form.state_id.state_id_number_hint_asterisks'), '*'],
+           [t('in_person_proofing.form.state_id.state_id_number_hint_dashes'), '-', true],
+         ].each do |text, symbol, last| %>
+         <span class="usa-sr-only"><%= text %><%= ',' if !last %></span>
+         <span aria-hidden="true"><%= symbol %></span>
+      <% end %>
+    <% end %>
+
+    <% state_id_number_hint = capture do %>
+      <% [
+        [:default, state_id_number_hint_default],
+        ['TX', t('in_person_proofing.form.state_id.state_id_number_texas_hint')],
```

Comment:
my concern here is how expansive this may get as we add more states. That was part of my thinking around adding the switch statement in the `state-guidance` file - to sort of separate out the strings in the switch to help maintain readability with the addition of more state specific text.

Reply:
There's still only one place to add new states, and it's this template. So I think in terms of code changes needed to add new states, it's about the same in the end?